js/icons/src/Warning.svelte

```diff
@@ -0,0 +1,18 @@
+<svg
+	fill="none"
+	stroke="currentColor"
+	stroke-width="2"
+	viewBox="0 0 24 24"
+	width="33"
+	height="33"
```

Comment:
Can we make this consistent with other icons? Other icons have their width + height set too 100% and the size is controlled by the parent.

Same for the other two icons.